modules/environment/aws/eks/main.tf

```diff
@@ -9,7 +9,7 @@ systemctl enable amazon-ssm-agent
 
 echo '{"registry-mirrors": [${ var.docker_registry_mirror != "" ? format("\"%s\"", var.docker_registry_mirror) : ""}]}' | cat /etc/docker/daemon.json - | jq -s '.[0] * .[1]' > /tmp/daemon.json
 mv /tmp/daemon.json /etc/docker/daemon.json
-systemctl reload docker
+systemctl restart docker
```

Comment:
let's revert this change, we want this to be changed in the upcoming PR to upgrade the LEAD clusters to 1.17